Ifzan/binary.js

```diff
@@ -0,0 +1,31 @@
+function binary (arr,x,y,find)
+{
+    if(Array.isArray(arr) !== true)
+    {
+    console.log('Invalid');
```

Comment:
see how you can throw an error after consoling it

Reply:
tried ` try
    {
    if(!Array.isArray(arr) || !arr.length) throw 'INVALID';
}
catch (err)
{ console.log(err);}`
but returns invalid too many times.